homework/RosmaryFermin/Problem 1 Animal Sort/src/nyc/c4q/RosmaryFC/Domestic.java

```diff
@@ -0,0 +1,13 @@
+package nyc.c4q.RosmaryFC;
+/**
+ * Created by c4q-rosmary on 5/2/15.
+ */
+
+//Create the Domestic interface.
+// This interface should include getters and setters for a name field, since pets tend to have their own names.
+public interface Domestic
+{
+     String getName();
+     String setName(String newName);
```

Comment:
setName should be return nothing as it just sets a name. So the signature method can be void setName(String newName).